src/main/interface/msp.c

```diff
@@ -1448,6 +1448,40 @@ static mspResult_e mspFcProcessOutCommandWithArg(uint8_t cmdMSP, sbuf_t *src, sb
             *mspPostProcessFn = mspRebootFn;
         }
 
+        break;
+    case MSP_MULTIPLE_MSP:
+        {
+            const uint8_t MAX_MSP_MULTIPLE_REQUESTS = 20;
+            uint8_t mspCmdBuf[MAX_MSP_MULTIPLE_REQUESTS];
+            uint8_t mspCmdSize[MAX_MSP_MULTIPLE_REQUESTS];
+            uint8_t maxMSPs = 0;
+            if(sbufBytesRemaining(src) == 0) return MSP_RESULT_ERROR;
```

Comment:
Please use blocks for all `if`, `for`, and `while` statements - this makes code easier to maintain.

Also, use one line per statement.

And coding convention in Betaflight is to use spaces after `if`, `for`, and `while`.